metadata/triggerMatch_update.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
just a name change to be consistent with the js snippet file naming to satisfy my OCD. It is technically not enforced in the metadata validator.